flepimop/gempyor_pkg/src/gempyor/utils.py

```diff
@@ -1187,3 +1188,83 @@ def _git_head(repository: Path) -> str:
         check=True,
     )
     return proc.stdout.decode().strip()
+
+
+def _git_checkout(repository: Path, branch: str) -> None:
```

Comment:
could return a CompletedProcess, right? might be useful to provide - definitely planning that for sync actions.

Reply:
Addressed in b8c410ac0e08c33b9509e4284a258bc19a8bb472